docs/notebooks/documents/getting_started.py

```diff
@@ -0,0 +1,159 @@
+# ---
+# jupyter:
+#   jupytext:
+#     text_representation:
+#       extension: .py
+#       format_name: percent
+#       format_version: '1.3'
+#       jupytext_version: 1.16.1
+#   kernelspec:
+#     display_name: Python 3 (ipykernel)
+#     language: python
+#     name: python3
+# ---
+
+# %% [markdown]
+# # Getting Started
+#
+# This document provides an overview for preparing to work with PDO contracts through the contract
+# docker container. It will walk through the necessary steps to set up a functional
+# configuration. It does assume that the reader is familiar with
+# [basic operation of Jupyter notebooks](https://jupytext.readthedocs.io/en/latest/).
+#
+# %% [markdown]
+# ## How Contracts Use Jupyter
+#
+# Many of the PDO contract families provide Jupyter notebooks to simplify interaction with contract
+# objects. Commonly, the notebooks provide a factory for instantiating a notebook for each new
+# contract from one of the contract specific templates. The templates provided generally include five
+# common sections:
+#
+# * Configure the Contract Object
+# * Initialize the Jupyter Interpreter and the PDO Environment
+# * Initialize the Contract Object Context
+# * Operate on the Contract
+# * View Contract Metadata
+#
+# *Configure the Contract Object*: Each notebook either loads an existing contract object from a
+# file (see the PDO documentation on contract collections) or creates a new contract object based on
+# an initial set of parameters. To create a new contract object, a factory will collect input that
+# is used to instantiate a template notebook with the set of required parameters.
+#
+# *Initialize the Jupyter Interpreter and PDO Environment*: To initialize the Jupyter interpreter,
+# the notebook loads the Juptyer helper module. This module imports all of the relevant PDO and
+# Jupyter IPython modules to simplify code in the notebook. It also defines several procedures that
+# are useful for initializing and interacting with the PDO environment.
+#
+# In addition, the Jupyter interpreter initialization configures an IPython extension that makes it
+# easier to provide code for multiple types of operations that can be performed on the contract
+# object. Specifically, the extension defines a magic keyword, `skip`, that takes a value or
+# expression that, if it evaluates to True, causes the code section to be skipped during notebook
+# execution.
+#
+# *Initialize the Contract Object Context*: The next section in the notebook creates a PDO context
+# for the contract object (see the PDO documentation for more information on contexts).  The context
```

Comment:
Same as above, if the documentation already exists, we should add a link here.

Reply:
agreed... and if you have a suggestion for how to do that I would happily incorporate. the closest that we have is to put in a ref to the pdo git hub repo. not my favorite solution.